apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -47,6 +47,15 @@ function showHomepage() {
         />
       )}
 
+      {(!isTeacher && showUiTips) && (
+        <UiTips
+          tips={
+            [
+              {type: "triggered", position: {top: 80, right: 15}, text: i18n.homepageUiTipAlreadyHome(), triggerId: "logo_home_link", arrowDirection: "up_corner"}
+            ]}
+        />
+      )}
```

Comment:
I'd split line 54 into multiple lines, as it's quite long.

Also, as an FYI the parens on lines 50/57 are unnecessary (though fine).